react/features/base/react/components/web/MultiSelectAutocomplete.js

```diff
@@ -134,7 +135,7 @@ class MultiSelectAutocomplete extends Component {
             isOpen: Boolean(this.state.items.length)
                 && Boolean(filterValue),
             items: filterValue ? this.state.items : []
-        }, this._sendQuery);
+        }, debounce(this._sendQuery, 200));
```

Comment:
I'm not sure this is being used correctly. debounce will return a debounced version of _sendQuery. The pattern I normally see used in such a case is to re-assign the returned method or set a new instance method. So the constructor would have something like "this._sendQuery = debounce(this.sendQuery)." But I could just be totally wrong because I haven't used the "debounce" package.